src/unix/linux_like/emscripten/mod.rs

```diff
@@ -1455,41 +1455,41 @@ f! {
     pub fn CPU_EQUAL(set1: &cpu_set_t, set2: &cpu_set_t) -> bool {
         set1.bits == set2.bits
     }
+}
+
+safe_f! {
+    pub {const} fn makedev(major: c_uint, minor: c_uint) -> crate::dev_t {
```

Comment:
Emscripten has actual functions for these in addition to macros so it would be possible to call out to them. But it would be a lot slower than this.